vignettes/getting-started.Rmd

```diff
@@ -74,6 +92,18 @@ You'll notice that this set up gives us a list of formatted data. This contains
 str(gimap_dataset)
 ```
 
+```{r}
+gimap_dataset <- run_qc(gimap_dataset, params = list(cell_line = "pgPEN_HeLa"), overwrite = TRUE)
+```
+
+The addition to this is that now a QC filter has been added to the `gimap_dataset` object, specifically found in `gimap_dataset$transformed_data$qc_filter`. It's a list of TRUE and FALSE. With TRUE meaning that it passes the filters and should be kept, and FALSE meaning that it didn't pass (right now the combined filter) and should not be kept for the analysis. Future work should let you select if you're using one, both, or neither of the filters.
```

Comment:
Nice! I like this!!

Reply:
Wondering if it should be its own thing and not under "transformed_data" though. I could see it being its own entry like gimap_dataset$qc_filter? What do you think?

Reply:
I went back and forth on where to put it and I put it there because I view everything in the `transformed_data` part of the data as having the same index/row sorting. Happy to move it if you think it should be its own entry

Reply:
I think its own entry seems good. I don't want the `transformed_data` part to become too big. We could envision a structure where each step a new section gets added. Aka `qc` for this section and then we have an `annotation` entry.

Reply:
Fixed in commit [fa2e9df](https://github.com/FredHutch/gimap/pull/16/commits/fa2e9df149923fd54cb84627d17448040e3bd2e4)